sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
-
+- Bug fix to address queries with VALUE MAX (or any other aggregate) that run into an issue if the query is executed on a container with at least one "empty" partition.
```

Comment:
NIT: In java and .Net we keep referencing the PR(s) in the changelog along with the description. We have received good customer feedback for it - so, my preference would be to start doing the same in python - better late than never :-)